metrics-tracing-context/src/lib.rs

```diff
@@ -9,20 +9,26 @@
 //! First, set up `tracing` and `metrics` crates:
 //!
 //! ```rust
-//! # use metrics_util::debugging::DebuggingRecorder;
-//! # use tracing_subscriber::Registry;
-//! use metrics_tracing_context::{MetricsLayer, TracingContextLayer};
+//! use metrics_exporter_prometheus::PrometheusBuilder;
+//! use metrics_tracing_context::MetricsLayer;
+//! use metrics_tracing_context::TracingContextLayer;
```

Comment:
```suggestion
//! use metrics_tracing_context::{MetricsLayer, TracingContextLayer};
```

Reply:
Hello @tobz.

I will fix the ci, if other changes are ok.

Reply:
I've added dependencies, but I still got the following error:

```
---- metrics-tracing-context/src/lib.rs - (line 11) stdout ----
error[E0277]: the trait bound `PrometheusRecorder: Recorder` is not satisfied
   --> metrics-tracing-context/src/lib.rs:32:30
    |
24  | metrics::set_global_recorder(recorder).unwrap();
    | ---------------------------- ^^^^^^^^ the trait `Recorder` is not implemented for `PrometheusRecorder`
  
```

I've failed to understand why as Recorder as implemented for PrometheusRecorder and all dependencies seems up to date.